packages/ingest/granule.js

```diff
@@ -132,8 +130,9 @@ class Discover {
       granuleId,
       dataType,
       version,
-      // Remove the granuleId property from each file
-      files: files.map((file) => omit(file, 'granuleId'))
+      // Retain only files with configs and omit granuleId from each file
+      files: files.filter((file) => this.fileTypeConfigForFile(file))
```

Comment:
I feel we still want the files which matches the granuleId, even if there is no configuration for the file.  Maybe I didn't understand the ticket correctly.  @laurenfrederick ?

Reply:
Ugh I am not sure. I am going back and forth. Maybe? @chuckwondo what would that break downstream?

Reply:
That's what I'm wondering too. I don't know what might bomb downstream if we include the files with the granule. I don't know if something downstream would expect every file in the granule to have an entry in the collection's `files` configuration. @markdboyd might have a better idea.

Reply:
@jennyhliu If we included `files` which didn't match anything in the `collection.files` configuration, then they wouldn't have any ultimate destination bucket to be placed in by `move-granules`. So they would just sit in the staging bucket they were copied to be `sync-granule`, unless I'm missing something, which I don't think we want

Reply:
Yeah good point. I think then we do not want them in there.

Reply:
Great. @jennyhliu since you asked the question, does that mean you were acting as reviewer for this PR?